machines/bleShare/scan/scanMachine.ts

```diff
@@ -612,6 +616,7 @@ export const scanMachine =
                 src: 'disconnect',
               },
             },
+            navigateToHistory: {},
```

Comment:
why this state is empty?

Reply:
I have compared same state with request machine and implemented at here.

Reply:
@Anil-kumar-Majji Comparing with request machine is one aspect. But do we really need it as we are not doing anything here?
It might be unnecessary in request machine as well and we copied same to scan machine